web_external/ImagesGallery/Listing/ImageWall.js

```diff
@@ -23,6 +23,19 @@ const ImageWall = View.extend({
             } else {
                 clickedImage.toggleSelected();
             }
+        },
+
+        'click .isic-images-imageWall-zoom': function (event) {
+            let imageId = $(event.currentTarget).parent().parent().data('imageId');
+            let clickedImage = this.images.get(imageId);
+
+            new ImageFullscreenWidget({ // eslint-disable-line no-new
+                el: $('#g-dialog-container'),
+                model: clickedImage,
+                parentView: this
+            }).render();
+
+            event.stopPropagation();
```

Comment:
I usually see this called first thing in the event handler, but it shouldn't matter when it's called.

Reply:
Yes, but the consistent style is a good point.